RevenueCatUI/CustomerCenter/Views/CompatibilityContentUnavailableView.swift

```diff
@@ -19,7 +19,7 @@ import SwiftUI
 #if os(iOS)
 
 /// A SwiftUI view for displaying a message about unavailable content
-@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(iOS 15.0, *)
 @available(macOS, unavailable)
 @available(tvOS, unavailable)
 @available(watchOS, unavailable)
```

Comment:
this availability was redundant and we were marking it first available and then unavailable for the same platforms